v1.1/show-jobs.md

```diff
@@ -4,35 +4,57 @@ summary: The SHOW JOBS statement lists all currently active schema changes and b
 toc: false
 ---
 
-<span class="version-tag">New in v1.1:</span> The `SHOW JOBS` [statement](sql-statements.html) lists details about currently active schema changes and backup/restore jobs, including:
+<span class="version-tag">New in v1.1:</span> The `SHOW JOBS` [statement](sql-statements.html) lists all of the types of long-running tasks your cluster has performed, including:
 
-- xx
-- xx
-- xx
+- Schema changes through `ALTER TABLE`
+- Enterprise [`BACKUP`](backup.html), [`RESTORE`](restore.html), and `IMPORT`
 
-These details let you...
+These details can help you understand the status of crucial tasks, as well as help you control them.
 
 <div id="toc"></div>
 
 ## Required Privileges
 
-XX
+By default, only the `root` user can execute `SHOW JOBS`.
 
 ## Synopsis
 
 {% include sql/{{ page.version.version }}/diagrams/show_jobs.html %}
 
 ## Response
 
-The following fields are returned for each session:
+The following fields are returned for each job:
 
 Field | Description
 ------|------------
+`id` | A unique ID to identify each job. This value is used if you want to control a job's status (e.g., canceling it).
+`type` | Describes the job as `SCHEMA CHANGE`, [`BACKUP`](backup.html), [`RESTORE`](restore.html), or `IMPORT`.
+`description` | The command that started the job.
+`username` | The user who started the job.
+`descriptor_ids` | An array of table IDs affected by the job. You can find which tables these IDs correlate to using the `table_id` column of `crdb_internal.tables`.
+`status` | Describes the job's current state as `pending`, `running`, `paused`, `failed`, `succeeded`, or `canceled`.
+`created` | The `TIMESTAMP` when the job was created.
+`started` | The `TIMESTAMP` when the job began running first.
+`finished` | The `TIMESTAMP` when the job was `succeeded`, `failed`, or `canceled`.
+`modified` | The `TIMESTAMP` when the job had anything modified.
+`fraction_completed` | The fraction (between `0.00` and `1.00`) of the job that's been completed.
+`error` | If the job `failed`, the error generated by the failure.
 
 ## Examples
 
-XX
+### Show jobs
+
+~~~ sql
+> SHOW JOBS;
+~~~
+~~~
++----------------+---------+-------------------------------------------+...
+|       id       |  type   |               description                 |...
++----------------+---------+-------------------------------------------+...
+| 27536791415282 | RESTORE | RESTORE db.* FROM 'azure://backup/db/tbl' |...
++----------------+---------+-------------------------------------------+...
+~~~
 
```

Comment:
It might be helpful to show how you can filter jobs, too. E.g.:

```
SELECT * FROM [SHOW JOBS] WHERE type = 'RESTORE' AND status IN ('running', 'failed') ORDER BY created DESC;
```